insights/insightsclient/insightsclient.go

```diff
@@ -277,17 +281,18 @@ func responseBody(r *http.Response) string {
 
 var (
 	counterRequestSend = metrics.NewCounterVec(&metrics.CounterOpts{
-		Name: "insightsclient_request_send_total",
 		Help: "Tracks the number of metrics sends",
```

Comment:
Maybe we could improve this message. Suggestions?

Reply:
Counter for the number of uploads sent.